google/resource-snippets/cloudkms-v1/alternatives/krm/cloudkms.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kms.cnrm.cloud.google.com/v1beta1
+kind: KMSCryptoKey
+metadata:
+  labels:
+    foo: ${DEPLOYMENT?} # {"$kpt-set":"deployment"}
+  name: ${DEPLOYMENT?}-cryptokey # {"$kpt-set":"deployment-sub"}
+spec:
+  keyRingRef:
+    name: test-keyring
+  purpose: ENCRYPT_DECRYPT
+---
+apiVersion: kms.cnrm.cloud.google.com/v1beta1
+kind: KMSKeyRing
+metadata:
```

Comment:
Could you include the project-id in annotations (add to metadata) to match with the project in the `parent` field in the DM config?
`
annotations:
     cnrm.cloud.google.com/project-id: #{project-id-in-parent}
`

Reply:
https://cloud.google.com/config-connector/docs/reference/resource-docs/kms/kmskeyring

Reply:
It's using namespace to link project and the main reason for parent field is to have the region? since we have location define, is this necessary?